docs/pipelines.md

```diff
@@ -1569,9 +1569,9 @@ no `runAfter` can be specified in `finally` tasks.
 
 ## Using Custom Tasks
 
-We are promoting Custom Task from `v1alpha1` to `v1beta1`. Starting from `v0.43.0`, Pipeline Controller is able to create either `v1alpha1` or `v1beta1` Custom Task gated by a feature flag `custom-task-version`, defaulting to `v1alpha1`. You can set `custom-task-version` to `v1alpha1` or `v1beta1` to control which version to create.
+We are promoting Custom Task from `v1alpha1` to `v1beta1`. Starting from `v0.43.0`, Pipeline Controller is able to create either `v1alpha1` or `v1beta1` Custom Task gated by a feature flag `custom-task-version`, defaulting to `v1beta1`. You can set `custom-task-version` to `v1alpha1` or `v1beta1` to control which version to create.
 
-We'll switch the default value of `custom-task-version` to `v1beta1` in release v0.44.0, and remove the flag as well as the entire alpha version in release v0.47.0. See the [migration doc](migrating-v1alpha1.Run-to-v1beta1.CustomRun.md) for details.
+We'll remove the flag `custom-task-version` and the entire alpha Custom Task in release v0.47.0 (tracked in the issue #5837). See the [migration doc](migrating-v1alpha1.Run-to-v1beta1.CustomRun.md) for details.
```

Comment:
Curious if this would be inconsistent with what's in [`deprecation.md`](https://github.com/tektoncd/pipeline/blob/main/docs/deprecations.md)?

Reply:
Great point!

Reply:
Oh actually, this is an alpha feature, so we don't need to claim deprecation: https://github.com/tektoncd/pipeline/blob/main/api_compatibility_policy.md#alpha-crds

Reply:
We don't have to but it would still be useful, especially since there's such heavy usage of custom tasks (plus pipelineresources are also in that table)

Reply:
SG, updating!